lms/static/js/views/fields.js

```diff
@@ -30,12 +31,41 @@
             tagName: 'div',
 
             indicators: {
-                'canEdit': '<span class="icon fa fa-pencil message-can-edit" aria-hidden="true"></span><span class="sr">' + gettext("Editable") + '</span>', // jshint ignore:line
-                'error': '<span class="fa fa-exclamation-triangle message-error" aria-hidden="true"></span><span class="sr">' + gettext("Error") + '</span>', // jshint ignore:line
-                'validationError': '<span class="fa fa-exclamation-triangle message-validation-error" aria-hidden="true"></span><span class="sr">' + gettext("Validation Error") + '</span>', // jshint ignore:line
-                'inProgress': '<span class="fa fa-spinner fa-pulse message-in-progress" aria-hidden="true"></span><span class="sr">' + gettext("In Progress") + '</span>', // jshint ignore:line
-                'success': '<span class="fa fa-check message-success" aria-hidden="true"></span><span class="sr">' + gettext("Success") + '</span>', // jshint ignore:line
-                'plus': '<span class="fa fa-plus placeholder" aria-hidden="true"></span><span class="sr">' + gettext("Placeholder")+ '</span>' // jshint ignore:line
+                'canEdit': HtmlUtils.joinHtml(
+                    HtmlUtils.HTML('<span class="icon fa fa-pencil message-can-edit" aria-hidden="true"></span>'),
+                    HtmlUtils.HTML('<span class="sr">'),
+                    gettext("Editable"),
+                    HtmlUtils.HTML('</span>')
+                ),
```

Comment:
I just tried that. The `'<span class="icon fa fa-pencil message-can-edit" aria-hidden="true"></span><span class="sr">` and `</span>` have to be in `.HTML()` so it started to look messy with `.interpolateHtml()`.